jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java

```diff
@@ -351,7 +357,8 @@ public void test_resolverFields_Set_null() throws Exception {
         };
     }
 
-    @Test(dataProvider="formatWithZoneWithChronology")
+    @Test
```

Comment:
If this test is commented out, dupe the previous comment. That way, it's obvious commenting out both tests is what you intended.